django-transparent/docker-compose.django-transparent.yml

```
@@ -11,7 +11,7 @@ services:
         # We do not need a network to generate keys at all
         network_mode: "none"
         environment:
-            # INSECURE!!! You MUST define your own ACRA_CONNECTOR_MASTER_KEY
+            # INSECURE!!! You MUST define your own ACRA_(SERVER|CONNECTOR)_MASTER_KEY
```

i got your idea, but i think we shouldn't make users life more complicated, let's help them to understand what we want from them.

```suggestion
            # INSECURE!!! You MUST define your own ACRA_SERVER_MASTER_KEY and ACRA_CONNECTOR_MASTER_KEY
```